src/test/java/core/basesyntax/HelloWorldTest.java

```diff
@@ -1,8 +1,84 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
 public class HelloWorldTest {
```

Comment:
The test class is named `HelloWorldTest`, which does not reflect the purpose of the tests. Rename it to something more specific to the functionality being tested, like `RegistrationServiceImplTest`.